### DISCUSSION TITLE
Hide category title

### DISCUSSION THREAD

**Discussion by patriciomh1965** [serenity-is/Serenity#5969] [General]
Hello Community

I am hiding the category title of a form. I have managed to hide it, but I have not been able to hide the title of the top bar, if someone knows how to do it I would appreciate it.
Here I leave the screen and the instruction :

XYZDialog.ts

this.element.find(".category-title:contains('Ubicación de Riesgo')").hide();

Sincerely.

![panta1](https://user-images.githubusercontent.com/42218266/133178530-f02fb222-4685-4b7d-80cb-b20ca0b6cd53.png)
![panta2](https://user-images.githubusercontent.com/42218266/133178543-6a9f1af4-85b0-4104-90a4-48847206a6aa.png)

**Comment by choss21** [comment]
Hi
```
let categoryAnchor = this.element.find(".category-title:contains('Ubicación de Riesgo')").find('.category-anchor');
let categoryAnchorName = categoryAnchor.attr('name');
categoryAnchor.closest('.s-PropertyGrid').find(`a[href='#${categoryAnchorName}']`).hide();
```
 
Use Serenity.Extra for more information https://github.com/dfaruque/Serenity.Extra/blob/fb0ee3272fd0fc90062f5f6bcdbd5b789659165a/SerExtraCore/SerExtraCore.Web/Modules/_Ext/_q/_q.editor-utils.ts


reguards

**Reply by patriciomh1965** [reply]
It worked correctly for me

Thank you very much dear

Atte.
